### PR TITLE
Aaron Negrete Caudillo

### PR DESCRIPTION
**Pull request by 1aaron**

### CODE REVIEW

**Comment by danielaocanto** [comment]
Done  ! Gracias Aaron

File: .all_unittest.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def test_20_primes(self):
             )
             self.assertEqual(
                 no_primes_result,
-                True,
+                False,
```
Comment (moylop260) [inline .all_unittest.py]: @1aaron Could you revert this change and fix the real method, please?

**Comment by 1aaron** [comment]
I thought that was an error to take care of... Ok, i'll revert it

**Comment by 1aaron** [comment]
done https://github.com/Vauxoo/vauxoo-applicant/pull/106/commits/b9aa8c770616eb2b24574984b20bc11a0662d42e
